### ISSUE TITLE
(Gutter): (Accessbility)Expand collapse buttons not accessible via tab (Read only mode)

### ISSUE THREAD

**Issue by thisisjasmeet** [ajaxorg/ace#5125]
### Describe the bug

Expand collapse buttons are not accessible via keyboard navigation

### Expected Behavior

user should be able to expand/collapse  rows using keyboard actions.

### Current Behavior

user is not able to focus on gutter icons using keyboard navigation

### Reproduction Steps

1) Open editor in read only mode
2) press tab to to access gutter expand/collapse buttons
Currently its not possible to do so

### Ace Version / Browser / OS / Keyboard layout

"react-ace": "10.1.0"

**Comment by nightwing** [comment]
It is already possible to use the F2 key (the toggleFoldWidget) to collapse/expand the code

**Comment by akoreman** [comment]
This was added as part of PR https://github.com/ajaxorg/ace/pull/5146 and will be included in the next Ace release.

**Comment by akoreman** [comment]
[Ace 1.20.0](https://github.com/ajaxorg/ace/releases/tag/v1.20.0) has been released which includes this change.
